cleanrl_utils/tuner.py

```diff
@@ -26,22 +26,42 @@ def __init__(
         self,
         script: str,
         metric: str,
-        target_scores: dict[str, list[int]],
-        params_fn: Callable[[optuna.Trial], dict],
+        target_scores: Dict[str, Optional[List[float]]],
+        params_fn: Callable[[optuna.Trial], Dict],
         direction: str = "maximize",
+        aggregation_type: str = "average",
         metric_last_n_average_window: int = 50,
-        pruner: optuna.pruners.BasePruner = None,
+        sampler: Optional[optuna.samplers.BaseSampler] = None,
+        pruner: Optional[optuna.pruners.BasePruner] = None,
         storage: str = "sqlite:///cleanrl_hpopt.db",
         study_name: str = "",
-        wandb_kwargs: dict[str, any] = {},
+        wandb_kwargs: Dict[str, any] = {},
     ) -> None:
         self.script = script
         self.metric = metric
         self.target_scores = target_scores
+        if len(self.target_scores) > 1:
+            if self.target_scores.values()[0] is None:
```

Comment:
Why not use as follows
```python
if None in self.target_scores.values():
```
Users can act smart by only providing scores for the first environment and `None` for all others.
E.g. In this case there will be no error but the issue of biased optimization still persists
```python
target_scores={
        "CartPole-v1": [0, 500],
        "Acrobot-v1": None,
    },
```
Also, the current code raises the following error.
```sh
TypeError: 'dict_values' object is not subscriptable
```

Reply:
Nice catch. Just changed it.